packages/server/src/validator/types.ts

```diff
@@ -1,7 +1,28 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 import * as Constants from './constants';
 import { Validator } from '.';
+import { ITypeValidation } from '../types/validator/types';
 
-export const TYPES = {
+export const TYPES: { [key: string]: ITypeValidation } = {
```

Comment:
Just a small nit: perhaps we could come up with a more descriptive name for this object instead of just `TYPES`, as that might be a bit too generic. Maybe something related to the Relay or the validation process—or well.. we could just keep it as `TYPES` I guess, lol

Reply:
If you have an idea of a better name, I’m open to hearing it 😄